_posts/2016-06-23-census-centrality.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  "Crowdsensing the Census - Exploring Street Network"
+date:   2016-07-20
+author: Carlos Espino
+---
+
+Last week we wanted to extract more information from Milan´s OSM data, so we performed visualization and analysis of its street network. We aim to find a relationship between the street connectivity and social deprivation. We also wanted to further investigate how this relationship differs from that of Milan in a city belonging to a developing country, particularly Mexico City.
+
+In order to compute connectedness of the street network, we tried the two following graph centrality measures: 
+
++ _Betweenness centrality_ measures the number of shortest paths between any given pair of nodes that pass through an individual node.  
```

Comment:
Here again, I would re-iterate: "... the number of shortest paths (streets, or edges) between any given pair of nodes (intersections)..."